src/pal/src/misc/corefx.cpp

```diff
@@ -346,3 +348,68 @@ ForkAndExecProcess(
     return success ? 0 : -1;
 }
 
+void CopyStat(struct corefx_stat* dst, const struct stat* src)
+{
+    dst->mode = src->st_mode;
```

Comment:
I will remove it then, I would like to limit the sorts of things we expose unless we are sure we are going to need it.